tmc-langs-rust/src/main/java/fi/helsinki/cs/tmc/langs/rust/util/Constants.java

```diff
@@ -0,0 +1,10 @@
+
+package fi.helsinki.cs.tmc.langs.rust.util;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+public class Constants {
```

Comment:
Could this be package private and final? Also consider renaming this to `CargoConstants`.